### DISCUSSION TITLE
Disabling anti bot

### DISCUSSION THREAD

**Discussion by WuzzyLV** [games647/FastLogin#688] [Q&A] [answered]
Is it possible to disable this feature, since moving to a different host it often enables for a short amount of time after restarting.

**Comment by TuxCoding** [comment, marked as answer]
No at the moment. However, you can set the expire time low and connection limit high. A relative high number should be enough.

**Reply by WuzzyLV** [reply]
It might be because of my host, no idea why tho I already set the connections to 6000 and expire to 1, but whenever I restart my proxy, and when I try to join straight after it starts the antibot security thing enables and it disables after a minute or 2. And it is not like there are 100 people trying to join straight after it launches its literally just me and it still activates.

**Reply by TuxCoding** [reply]
I could add some debug messages to verify when it adds entries.

**Reply by WuzzyLV** [reply]
That would be fantastic

**Reply by TuxCoding** [reply]
Found the logic bug. I will upload a fix.
